setup.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+setup(
+    name='da_vinci',
+    version='0.0.0',
+    description='DaVinci Depth Estimation + Stereo View Prediction',
+    author='',
+    author_email='',
+    # REPLACE WITH YOUR OWN GITHUB PROJECT LINK
+    url='https://github.com/annikabrundyn/da_vinci',
+    install_requires=['pytorch-lightning'],
+    packages=find_packages(),
+)
```

Comment:
setup.py file needed to install project as python package

to install, run: pip install -e . 
(added to readme)